dandi/metadata/util.py

```diff
@@ -840,6 +840,36 @@ class Neurodatum(TypedDict):
         "technique": None,
         "approach": "optogenetic approach",
     },
+    "PoseEstimationSeries": {
+        "module": "ndx-pose",
```

Comment:
@rly - the module is what gets attached at the end of the filename. so this is where the classification would go in, not which extension it is from. should this be `behavior` instead? see earlier for some examples.